Core.ApplicationServices/Organizations/OrganizationRightsService.cs

```diff
@@ -75,6 +90,92 @@ public class OrganizationRightsService : IOrganizationRightsService
             return RemoveRight(right);
         }
 
+        public Maybe<OperationError> RemoveUnitRightsByIds(int organizationId, IEnumerable<int> rightIds)
+        {
+            using var transaction = _transactionManager.Begin();
+
+            var result = CanModifyOrganization(organizationId);
+            if (result.HasValue)
+            {
+                return result;
+            }
+
+            var rightsToDelete = new List<OrganizationUnitRight>();
+            foreach (var rightId in rightIds)
```

Comment:
If I provide a list of any right from any org, then I would get away with it as long as I had access to the organization in parameter 1.. you are not scoping your actions..

The right thing would be to get the organization unit FROM the organization or with the org id as condition,,,,

Then you would know that it came from the same organization....

Also... we have a `IOrganizationUnitRepository`.. why not add a "Result<OrganizationUnit,OperationError> GetOrganizationUnit(int organizationId, int organizationUnitId)".. that would then ensure to do it right